tools/generate_validation_tolerances.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+This script reproduces the computation used for generating the validation tolerances in csrc/validator_utils.h:
+https://github.com/NVIDIA/Fuser/blob/892b7ac5646b3b873f13f2b2ea1db15fa9b9effb/csrc/validator_utils.h#L25-L46
+
+The computation (randn + mul + sum) is repeated a large number of times.
+The base tolerance is the max error seen when comparing the desired precision type with the next higher precision.
+The final tolerances are computed by adding a safety factor of 3/4 to this base tolerance.
+
+The output of this file is a .npy file corresponding to each dtype, containing a {size: base_tolerances} dict.
```

Comment:
I prefered storing it as a file, since I was experiementing with 1) Safety factors 2) Graphs comparing different devices and existing constants. This is an experimentation script, so I much prefer storing the data. The script also runs for a large time -- so I do not work with it interactively.